Formula/sbcl.rb

```diff
@@ -12,22 +12,11 @@ class Sbcl < Formula
     sha256 cellar: :any_skip_relocation, mojave:   "a8eadb8a7b8a092995d0f29f4b68cf5450c070dd41bdf3f9f15e4907b516d48b"
   end
 
+  depends_on "ecl" => :build
   depends_on arch: :x86_64
```

Comment:
We can put this `depends_on` in a `stable` block, so that `brew install --HEAD sbcl` works without any modifications to the formula on ARM. `grep 'stable do'` for an example.

Reply:
Done, thanks.